### PR TITLE
DOCS-2500 Alerting section link updates

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Update links for Alerting section restructure https://github.com/DataDog/documentation/pull/11576

### Motivation
Alerting section restructure

### Preview
https://docs-staging.datadoghq.com/ruth/docs-2500/monitors/

### Additional Notes
N/A

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/create/types/apm.md [hunk 1]
```diff
@@ -3,18 +3,18 @@ title: APM Monitor
 kind: documentation
 description: "Compare an APM metric to a user defined threshold"
 aliases:
-  - /monitors/monitor_types/app_analytics
-  - /monitors/monitor_types/trace_search
+  - /monitors/create/types/app_analytics
+  - /monitors/create/types/trace_search
   - /tracing/guide/resource_monitor/
-  - /monitors/monitor_types/apm
+  - /monitors/create/types/apm
```
Comment (kayayarai) [inline content/en/monitors/create/types/apm.md:9]: I think for aliases you want to keep the old ones (and maybe add the new ones? but definitely keep the old ones)

File: content/en/monitors/create/types/composite.md [hunk 2]
```diff
@@ -3,16 +3,16 @@ title: Composite Monitor
 kind: documentation
 aliases:
     - /guides/composite_monitors
-    - /monitors/monitor_types/composite
+    - /monitors/create/types/composite
```
Comment (kayayarai) [inline content/en/monitors/create/types/composite.md:6]: keep the old one too

File: content/en/monitors/create/types/custom_check.md [hunk 3]
```diff
@@ -3,15 +3,15 @@ title: Custom Check Monitor
 kind: documentation
 description: "Monitor status of arbitrary custom checks."
 aliases :
-    - /monitors/monitor_types/custom_check
+    - /monitors/create/types/custom_check
```
Comment (kayayarai) [inline content/en/monitors/create/types/custom_check.md:6]: keep the old one too

**Review by kayayarai — CHANGES_REQUESTED** [review]
The changes to links look fine throughout (I skimmed), but the changes to aliases should at least keep the old aliases. We don't want to lose redirects from the old structure. I'm not sure that you need aliases from old file names with new folder names, though. (I commented on a few, but then stopped).

**Comment by ruthnaebeck** [comment]
> The changes to links look fine throughout (I skimmed), but the changes to aliases should at least keep the old aliases. We don't want to lose redirects from the old structure. I'm not sure that you need aliases from old file names with new folder names, though. (I commented on a few, but then stopped).

Fixed, thanks!

**Pull request merged**